recipes/tracktable/conda_build_config.yaml

```diff
@@ -0,0 +1,47 @@
+# CONDA_BUILD_SYSROOT:            # [osx]
```

Comment:
This should all be pulled from the global pinning and not be manually added here. You should delete the file entriely.

Reply:
conda_build_config.yaml has been deleted